index.php

```diff
@@ -15,11 +15,25 @@
 <body>
     <main>
         <h1>BEHOLD!! Some deities and stuff:</h1>
-        <div>
+        <div class="displayDeities">
             <?php foreach ($deities as $deity) {
                 echo displayDeity($deity); 
             } ?>
         </div>
+        <div class="addNewDeityForm">
+            <h3>Can't see your favourite deity? No problem! Add them with this form:</h3>
```

Comment:
the structure of the headings with relation to each other doesn't quite make sense, you will have to change them around (as discussed)